app/domain/ptime/assign_employee_ids.rb

```diff
@@ -0,0 +1,80 @@
+# This script will assign each person the corresponding employee ID from PuzzleTime
+module Ptime
+  class AssignEmployeeIds
+    include PtimeHelper
+
+    # rubocop:disable Rails/Output
+    def run(should_map: false)
+      puts 'Notice this is a dry run and mapping will not happen!' unless should_map
+      fetch_data
+
+      @ptime_employees_by_provider.each do |provider, provider_employees|
+        @provider = provider
+        @provider_employees = provider_employees
+
+        puts '-' * 100
```

Comment:
You do this several times, sometimes `* 80` sometimes `* 100`
How about defining a constant named `OUTPUT_SEPARATOR` and using that instead ? :grimacing: